Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -671,8 +690,89 @@ private class PhotonActionSheetCell: UITableViewCell {
             toggleSwitch.accessibilityIdentifier = action.isEnabled ? "enabled" : "disabled"
             toggleSwitch.image = image
             stackView.addArrangedSubview(toggleSwitch)
+        case .Sync:
+            if let manager = syncManager {
+                if syncButton == nil {
+                    let button = SyncMenuButton(with: manager)
+                    stackView.addArrangedSubview(button)
+                    syncButton = button
+                    syncButton?.contentHorizontalAlignment = .right
+                    syncButton?.snp.makeConstraints { make in
+                        make.size.equalTo(40)
+                        make.height.equalTo(40)
+                    }
+                }
+                syncButton?.updateAnimations()
+                let padding = PhotonActionSheetCell.Padding
+                stackView.snp.remakeConstraints { make in
+                    make.edges.equalTo(contentView).inset(UIEdgeInsets(top: 0, left: padding, bottom: 0, right: padding))
```

Comment:
ASIDE: I am digging the non-snp way lately (because I can prototype stackview layout in Interface Builder using stackview properties and layoutMargins, and then when manually coded it will match IB behaviour closely):
```
stackview.layoutMargins = UIEdgeInsets(top: , left: , bottom: , right: )
stackview.isLayoutMarginsRelativeArrangement = true
```
Doesn't really apply to PhotonActionSheet as it doesn't need prototyping in IB at this stage.